flows/vision-zero/cris_import/cris_import.py

```diff
@@ -546,57 +610,184 @@ def align_records(typed_token, dry_run):
             else:
                 # An insert is always just an vanilla insert, as there is not a pair of records to compare.
                 # Produce the SQL which creates a new VZDB record from a query of the imported data
-                insert_statement = util.form_insert_statement(output_map, table, input_column_names, import_key_sql, DB_IMPORT_SCHEMA)
+                insert_statement = util.form_insert_statement(output_map, table, input_column_names, import_key_sql, map_state["import_schema"])
                 logger.info(f"Executing insert in {output_map[table]} for where " + record_key_sql)
 
                 # Execute the insert statement
                 util.try_statement(pg, output_map, table, record_key_sql, insert_statement, dry_run)
 
     # fmt: on
-    return True
+    return map_state
+
+
+@task(
+    name="Group CSVs into logical groups",
+)
+def group_csvs_into_logical_groups(extracted_archives, dry_run):
```

Comment:
Curious what your thoughts about having a known list of schemas (2018, 2023) that we check for here, and, if there is a new one, then raise an error? I don't imagine a new one coming along soon, but who knows!? Could be a way to catch a change in the future.

It also could be good to drop a comment of the known csv filename like `extract_2023_20230419092605_crash_20230418_HAYSTRAVISWILLIAMSON.csv` somewhere as an example.

Reply:
Yea, I think that is a super good idea. I'm going to spin off an issue for that right now.